util/util.go

```diff
@@ -347,6 +347,20 @@ func SplitStringToMap(str, separator string) map[string]struct{} {
 	return ret
 }
 
+// AutoCorrectName converts name to lowercase, and
+// correct overlength name by replaces the name suffix
+// with 8 char from its checksum to ensure uniqueness.
+func AutoCorrectName(name string, maxLength int) string {
```

Comment:
Add a log if the name is different after the auto correction

Reply:
Addressed.